mesheryctl/internal/cli/root/system/error.go

```diff
@@ -42,6 +42,15 @@ const (
 	ErrInitPortForwardCode               = "1047"
 	ErrRunPortForwardCode                = "1048"
 	ErrFailedGetEphemeralPortCode        = "1049"
+	ErrProcessingConfigCode              = "1050"
+	ErrCreatingConfigFileCode            = "1051"
+	ErrAddingTokenToConfigCode           = "1052"
+	ErrAddingContextToConfigCode         = "1053"
+	ErrUnmarshallingConfigFileCode       = "1054"
+	ErrGettingRequestContextCode         = "1055"
+	ErrInvalidAPIResponseCode            = "1056"
+	ErrUnmarshallingAPIDataCode          = "1057"
+	ErrConnectingToServerCode            = "1058"
```

Comment:
@zakisk if you look at the pr's #8238 and #8256, most of these error codes and functions you wrote here  are already implemented in the pr's i just listed  , the error functions in the `utils package` are for general use for all the packages in mesheryctl for returning errors,  you may have to remove some of these here to avoid redudancy . 
Only create the ones applicable to the `system` package, or you can add more to the utils package as far as it hasn't been implemented already in the pr's  i mentioned.
please look through the pr's .

Reply:
@Philip-21 as PR #8256 contains error code those I've created in system/error.go but this PR isn't meged yet, am I supposed to wait for its merge? or anything else better you suggest me?

Reply:
you are supposed to wait for both pr's to merge